src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -24,6 +24,10 @@ describe('ReactElement', function() {
   beforeEach(function() {
     require('mock-modules').dumpCache();
 
+    // Delete the native Symbol if we have one to ensure we test the
+    // unpolyfilled environment.
+    delete global.Symbol;
```

Comment:
Do you think it's worth requiring a symbol polyfill similar to how an ES5 one is required? Seems kinda dangerous to allow older browsers to be vulnerable.

On Thu, Sep 10, 2015 at 6:12 PM, Christoph Pojer <notifications@github.com>
wrote:

>> @@ -24,6 +24,10 @@ describe('ReactElement', function() {
>>    beforeEach(function() {
>>      require('mock-modules').dumpCache();
>>  
>> +    // Delete the native Symbol if we have one to ensure we test the
>> +    // unpolyfilled environment.
>> +    delete global.Symbol;
> ohhh.. right, it is from the engine. Let's see if this test will fail with newer versions of node! Don't you still use node 0.10 for jest right now? If yes, that doesn't have Symbol, does it?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/facebook/react/pull/4832/files#r39185824